routes/api.php

```diff
@@ -1,17 +1,11 @@
 <?php
 
-use App\Domains\Settings\ManageUsers\Api\UserController;
+use App\Domains\Settings\ManageUsers\Api\Controllers\UserController;
 use Illuminate\Support\Facades\Route;
 
-/*
-|--------------------------------------------------------------------------
-| API Routes
-|--------------------------------------------------------------------------
-|
-| Here is where you can register API routes for your application. These
-| routes are loaded by the RouteServiceProvider within a group which
-| is assigned the "api" middleware group. Enjoy building your API!
-|
-*/
-
 Route::middleware(['auth:sanctum', 'abilities:read'])->get('/user', UserController::class);
+
+Route::middleware(['auth:sanctum', 'abilities:read'])->group(function () {
```

Comment:
The `Route::middleware(['auth:sanctum', 'abilities:read'])` must be grouped with the previous line. Actually the `abilities:read` middleware should be define inside the controller itself, it will be easier.

Reply:
Done

Reply:
Actually not that done.

If we put the middleware in the controller, we have to put in the constructor. However, we also need to add a middleware in the constructor class of the ApiController parent controller, which checks the limit parameter and apply it to the query.

If we do that, it doesn't work, since we can't override the constructors in both parent and child in PHP.

SOOOO... we need to add the `abilities` in the routes :-/